core/src/smartcontracts/isi/query.rs

```diff
@@ -103,6 +103,7 @@ impl<W: WorldTrait> ValidQuery<W> for QueryBox {
             FindAccountById(query) => query.execute_into_value(wsv),
             FindAccountsByName(query) => query.execute_into_value(wsv),
             FindAccountsByDomainId(query) => query.execute_into_value(wsv),
+            FindAccountsWithAsset(query) => query.execute_into_value(wsv),
```

Comment:
not really a big deal, but a small declarative macro could be implemented for this long enum

Reply:
There's a crate that can forward implementations to enum variants. It can also remove the need for manually adding the `enum` variant.

Reply:
Create new issue?

Reply:
I'll do that.